Sources/NetworkProtection/NetworkProtectionDeviceManager.swift

```diff
@@ -285,21 +277,31 @@ public actor NetworkProtectionDeviceManager: NetworkProtectionDeviceManagement {
         let dns: [DNSServer]
         switch dnsSettings {
         case .default:
-            dns = [DNSServer(address: server.serverInfo.internalIP)]
+            dns = [DNSServer(address: server.serverInfo.internalIP.ipAddress)]
         case .custom(let servers):
             dns = servers
                 .compactMap { IPv4Address($0) }
                 .map { DNSServer(address: $0) }
         }
 
-        let interface = interfaceConfiguration(privateKey: interfacePrivateKey,
-                                               addressRange: interfaceAddressRange,
-                                               includedRoutes: includedRoutes,
-                                               excludedRoutes: excludedRoutes,
-                                               dns: dns,
-                                               isKillSwitchEnabled: isKillSwitchEnabled)
+        let routingTableResolver = VPNRoutingTableResolver(
```

Comment:
This class takes care of calculating the appropriate routes.  Makes it really easy to write unit tests as well.

I'll try to get some in before we merge this.